### ISSUE TITLE
VO Bot + All Browsers - Missing output in tests/toggle-button

### ISSUE THREAD

**Issue by ChrisC** [w3c/aria-at-app#1324]
There's consistent missing output in tests 21 and 24 of `tests/toggle-button` happening in Voiceover across all browsers. Here's an example in Chrome:
https://github.com/bocoup/aria-at-gh-actions-helper/wiki/20250130-Consistency-Report#test-number-21-12

Compare these results to NVDA output:
https://github.com/bocoup/aria-at-gh-actions-helper/wiki/20250130-Consistency-Report#test-number-21-11

**Comment by ChrisC** [comment]
This is also being reported by manual testers in the published reports: https://aria-at.w3.org/report/163626/targets/312

Closing this since it's not actually an Automation or VO Bot issue.
